### PR TITLE
Add iquery and --human

### PR DESCRIPTION
**Pull request by lldelisle**
Hi,
I added some --human flags. I also changed the `--nice` to `--human` for consistency. 
I filled some fields and flags.
How can I update the doc? and the big file gxadmin?

Thanks

### CODE REVIEW

File: parts/22-query.sh [hunk 1]
```diff
@@ -981,15 +1002,23 @@ query_monthly-data(){ ## [year]: Number of active users per month, running jobs
 		     2018-02-01 | 18 TB
 		     2018-01-01 | 16 TB
 	EOF
-
+	size="sum(coalesce(dataset.total_size, dataset.file_size, 0))"
 	if (( $# > 0 )); then
-		where="WHERE date_trunc('year', dataset.create_time AT TIME ZONE 'UTC') = '$1-01-01'::date"
+		for args in "$@"; do
```
Comment (hexylena) [inline parts/22-query.sh]: normally here I'd done something like checking if $2 was set and behaving around that, this is much better obviously. I had intended to add `getopt` parsing or similar but had not had time or energy for it yet, so it makes sense to add these one-off argument parsing bits.

**Comment by hexylena** [comment]
Hi @lldelisle just run `make` to update the documentation :) then this looks good! Can you document this in the changelog as well please? (And add yourself to the readme as one of the contributors, if you wish?)

**Comment by lldelisle** [comment]
I updated the doc. For the moment, I consider my contribution too small to be considered as contributor... Thanks for this tool suite this is very useful.

**Comment by hexylena** [comment]
So just as FYI: the `gxadmin` script wasn't meant to be included in the repository any more (you did nothing wrong here, my fault.) I've removed it, so your contributions will show up in the next tagged release of the project, findable on the releases page.

If you're using the ansible role it will download the master branch and 'compile' that, so no change needed.

**Comment by lldelisle** [comment]
Oups I forgot the changelog. Thanks...

**Pull request merged**